docs/development/development-environment-docker/README.md

```diff
@@ -21,6 +21,16 @@ bin/compose start
 
 Once the containers are done booting you can access the application under http://localhost:3000.
 
+### Tests
+
+You can run tests using `bin/compose rspec`. You can run specific tests too. For instance:
+
+```
+bin/compose rspec spec/features/work_package_show_spec.rb
+```
+
+***
```

Comment:
I'd place this line above the section about tests. 

Additionally, I'd take the chance to rename the heading `Setup` to something like `Setup (Step by step)` to be easily distinguishable from `Quickstart`